assessment.md

```diff
@@ -32,15 +32,124 @@
+  messages provide more detail but still may leave room for confusion, and the
+  student engages in a team review and discussion session.
+  * Students should receive an '😎' if all work is done in the team branch,
+  commit messages provide extensive depth regarding changes, and the student
+  actively engages in and/or initiates a team review and discussion.
+
 * Using the GitHub issue tracker
+  * 😞 = do not use the issue tracker
+  * 😕 = occasionally use the issue tracker, issue descriptions are confusing
+  * 😐 = use the issue tracker, but will still occasionally use slack or
+  other communication, issue descriptions occasionally confusing
+  * 😃 = use the issue tracker, issue descriptions not as in depth as they
+  should be
+  * 😎 = use the issue tracker, issue descriptions are in depth and clear
+
 * Assigning team members to issues in the GitHub issue tracker
+
+  * :disappointed: = No team member was assigned to issue.
+  * :confused: = Team member was assigned to issue but no response was given.
+  * :neutral_face: = Team member was assigned to issue and team member tried to
+    assist but no change was made.
+  * :smiley: = Team member was assigned to issue and issue was resolved.
+  * :sunglasses: = Team member was assigned to issue and issue was resolved and
+    understood.
+
 * Creating and assigning labels to issues in the GitHub issue tracker
+  * :disappointed: = Doing nothing.
+  * :confused:= Assigning but not clarifying tasks.
+  * :neutral_face: = Assigned tasks that are clear but are useless.
+  * :smiley: = Assigning tasks that are clear and beneficial to the group.
+  * :sunglasses: = Assigning tasks that are clear, beneficial and given in a
+     timely manner.
+
 * Creating and using a repository branch
+
+  * :disappointed: = No branches (other than master) are used, and commits
+    are made directly to the master branch.
+  * :confused: = A branch is used, but its name is not relevant to its
+    purpose or its function is unnecessary.
+  * :neutral_face: = Branch name is relevant to its function and its
+    function is necessary.
+  * :smiley: = Branch is properly created and necessary, and a pull
+    request is created once work is completed on the branch.
+  * :sunglasses: = Branch is properly created, and a pull request is
+    created once work is completed on the branch. Guidance is provided to
+    group members who need assistance, ensuring that use of branches is
+    consistent across team members.
+
+A software engineer demonstrates mastery of this technical skill at the
+:sunglasses: level by communicating with their team to determine a naming scheme
+for branches, and the plan is executed as agreed upon. The branch(es) being used
+should be named according to their role (i.e.
+team2-project-management-with-github), and once that role is complete/no longer
+necessary, a pull request should be created to merge the branch with the main
+branch.
```

Comment:
This section should be removed according to the markdown assessment template

Reply:
This specific issue (The presence of this section) is currently being addressed as an issue in the Markdown repository.